nin/backend/serve.js

```diff
@@ -10,17 +10,9 @@ var readDir = require('readdir');
 var socket = require('./socket');
 var watch = require('./watch');
 
-var publishing = false;
-
 
 var serve = function(projectPath, shouldRunHeadlessly) {
-
-  var genPath = p.join(projectPath, '/gen/');
-  mkdirp.sync(genPath);
```

Comment:
This used to create the `gen/` folder in a demo project's folder. We probably still want to do this, as other parts of nin assume that this directory exists.